src/components/AvatarGroup/AvatarGroup.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/components/Avatar/Avatar.scss";
+@import "../../styles/themes";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
+}
+
+.avatarContainer {
+  margin-left: -8px;
+  // TODO customize spacing
+  :hover {
+    z-index: 2;
+  }
+}
+
+.counterContainer {
+  background: var(--primary-background-hover-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -8px;
+  z-index: 1;
+
+  &:focus {
+    outline: none;
+    @include focus-style(50%);
+  }
+
+  &.small {
+    height: $avatar-size-small;
+    width: $avatar-size-small;
+    border-radius: $avatar-size-small / 2;
+  }
+
+  &.medium {
+    height: $avatar-size-medium;
+    width: $avatar-size-medium;
+    border-radius: $avatar-size-medium / 2;
+  }
+
+  &.large {
+    height: $avatar-size-large;
+    width: $avatar-size-large;
+    border-radius: $avatar-size-large / 2;
+  }
+
+  &.primary {
+    @include theme-prop(color, text-color-on-primary);
```

Comment:
please convert to use var(--text-color-on-primary), we are not using theme-prop anymore